src/cftlib/java/uk/gov/hmcts/probate/CftLibConfig.java

```diff
@@ -22,6 +22,12 @@ public void configure(CFTLib lib) throws Exception {
         lib.createProfile("probatesolicitortestorgtest1@gmail.com", "PROBATE", "Caveat", "Pending");
         lib.createProfile("probatecaseworker@gmail.com", "PROBATE", "GrantOfRepresentation", "Pending");
         lib.createProfile("probatecaseworker@gmail.com", "PROBATE", "Caveat", "Pending");
+        lib.createProfile("probatesolicitortestorgtest1@gmail.com", "PROBATE", "LegacySearch", "Pending");
+        lib.createProfile("probatesolicitortestorgtest1@gmail.com", "PROBATE", "Caveat", "Pending");
+        lib.createProfile("probatecaseworker@gmail.com", "PROBATE", "GrantOfRepresentation", "Pending");
+        lib.createProfile("probatecaseworker@gmail.com", "PROBATE", "Caveat", "Pending");
+        lib.createProfile("probatebackoffice@gmail.com", "PROBATE", "LegacySearch", "Pending");
```

Comment:
I dont think there is a Pending (default) state for LegacySearch case types

Reply:
I'll just take these out as for local non aat